src/Microsoft.Azure.WebJobs.Extensions.Redis/RedisExtensionConfigProvider.cs

```diff
@@ -53,14 +54,15 @@ public void Initialize(ExtensionConfigContext context)
             FluentBindingRule<RedisStreamTriggerAttribute> streamTriggerRule = context.AddBindingRule<RedisStreamTriggerAttribute>();
             streamTriggerRule.BindToTrigger(new RedisStreamTriggerBindingProvider(configuration, nameResolver, loggerFactory.CreateLogger("RedisStreamTrigger")));
 
-            FluentBindingRule<RedisAttribute> outputBindingRule = context.AddBindingRule<RedisAttribute>();
-            outputBindingRule.BindToCollector((attr) => new RedisAsyncCollector(GetOrCreateConnectionMultiplexer(attr), attr.Command, loggerFactory.CreateLogger("RedisOutputBinding")));
+            FluentBindingRule<RedisAttribute> bindingRule = context.AddBindingRule<RedisAttribute>();
+            bindingRule.BindToCollector((attr) => new RedisAsyncCollector(GetOrCreateConnectionMultiplexer(configuration, attr.ConnectionStringSetting), attr.Command, loggerFactory.CreateLogger("RedisOutputBinding")));
+            bindingRule.BindToInput<OpenType>(typeof(RedisConverter<>), configuration, nameResolver, loggerFactory.CreateLogger("RedisInputBinding"));
 #pragma warning restore CS0618
         }
 
-        internal IConnectionMultiplexer GetOrCreateConnectionMultiplexer(RedisAttribute attribute)
+        public static IConnectionMultiplexer GetOrCreateConnectionMultiplexer(IConfiguration configuration, string connectionStringSetting)
```

Comment:
Are we exposing the ConnectionMultiplexer (internal implementation details ) to customer code here?

Reply:
Reverted this back to internal static